docs/visual-basic/language-reference/data-types/byte-data-type.md

```diff
@@ -14,6 +14,7 @@ helpviewer_keywords:
 ms.assetid: eed44dff-eaee-4937-a89f-444e418e74f6
 author: "rpetrusha"
 ms.author: "ronpet"
+ms.manager: "wpickett"
```

Comment:
Wade is already the manager on the global settings. Otherwise, the metadata field is manager not ms.manager.